NanamiL3.java

```diff
@@ -0,0 +1,122 @@
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+// import packages to read user input from console
+
+public class NanamiL3 {
+    private static Task[] tasklist = new Task[100];
```

Comment:
Can consider avoiding magic numbers and use a named constant instead e.g. MAX_TASKS